modules/api/src/test/scala/vinyldns/api/domain/zone/ZoneServiceSpec.scala

```
@@ -95,23 +95,24 @@ class ZoneServiceSpec
       resultZone.name shouldBe zoneAuthorized.name
       resultZone.status shouldBe ZoneStatus.Syncing
       resultZone.connection shouldBe zoneAuthorized.connection
+      resultZone.shared shouldBe false
     }
 
-    "returns a ZoneAlreadyExists error if the zone exists" in {
+    "return a ZoneAlreadyExists error if the zone exists" in {
```

why "return" instead of "returns"?

---

That's the grammar aspect. `Creating Zones should return a ZoneAlreadyExists error if the zone exists`